docs/v.list

```diff
@@ -4,11 +4,11 @@
 
 <vars>
 
-    <var name="kotlinVersion" value="1.7.20" type="string"/>
```

Comment:
kotlinVersion and LatestUrl vars should be udpated only when the stable 1.8.0 is out